### ISSUE TITLE
Unstable initialisation of new machine example

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#368]
**Reported by dietmarw on 7 May 2010 08:18 UTC**
The following nice example is unfotunately a bit unstable when it come to initialisation:
`Examples.SynchronousInductionMachines.SMEE_LoadDump`

The tool manages to successfully simulate in the end but the model itself is very sensitive to changes. For example just removing the switch will cause the model to fail. Maybe this can be made a bit more stable?

Here the log I got when simulating the example:
```
Model: Modelica.Electrical.Machines.Examples.SynchronousInductionMachines.SMEE_LoadDump
... "dsin.txt" loading (dymosim input file)
The following error was detected at time: 0

Model error - division by zero: (1) / (sqrt(voltageQuasiRMSSensor.ToPolar1.u[1]^2+voltageQuasiRMSSensor.ToPolar1.u[2]^2)) = (1) / (0)

First evaluation failed for non-linear solver.

The following error was detected at time: 0

Model error - division by zero: (1) / (sqrt(voltageQuasiRMSSensor.ToPolar1.u[1]^2+voltageQuasiRMSSensor.ToPolar1.u[2]^2)) = (1) / (0)

Non-linear solver will attempt to handle this problem.
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/368

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 8 May 2010 08:53 UTC**
1. How should a load dump be realized without a switch?
2. No, the model runs even without the switch.
3. (sqrt(voltageQuasiRMSSensor.ToPolar1.u[1]<sup>2+voltageQuasiRMSSensor.ToPolar1.u[2]</sup>2)) 
   is exactly the output of the voltageQuasiRMSSensor, whose start value is set > 0.
   Why does the tool ignore that?
So, how make the initialization more stable?

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 8 May 2010 08:55 UTC**
Sorry for the bad formatting:
1. How should a load dump be realized without a switch?
2. No, the model runs even without the switch.
3. The variable(s) of the log
```
(sqrt(voltageQuasiRMSSensor.ToPolar1.u[1]2+voltageQuasiRMSSensor.ToPolar1.u[2]2))
```
    are exactly the output of the voltageQuasiRMSSensor, whose start value is set > 0.
    Why does the tool ignore that?

So, how make the initialization more stable?

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 8 May 2010 12:08 UTC**
Replying to [comment:2 AHaumer]:
> Sorry for the bad formatting:
> 1. How should a load dump be realized without a switch?
> 2. No, the model runs even without the switch.

Ah that was a misunderstanding, in my application I removed the switch and it didn't work. But at the same time I replaced the speed ramp with a constant speed value (wNominal). That was actually the reason for the model failing. So jerking the generator up to `wNominal` seems to be general problem (not necessarily for this example since here the ramp is used in any case). 

> 3. The variable(s) of the log
> ```
> (sqrt(voltageQuasiRMSSensor.ToPolar1.u[1]2+voltageQuasiRMSSensor.ToPolar1.u[2]2))
> ```
>     are exactly the output of the voltageQuasiRMSSensor, whose start value is set > 0.
>     Why does the tool ignore that?
> 
> So, how make the initialization more stable?

Good question...

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 8 Jul 2010 07:58 UTC**
resolved by c1f8faeff68b71216cfb2800cb44ffd6485ffbc4.

**Comment by modelica-trac-importer** [comment]
**Modified by AHaumer on 8 Jul 2010 17:43 UTC**
